core/jvm/src/main/scala/cats/effect/unsafe/IORuntimeCompanionPlatform.scala

```diff
@@ -104,6 +106,13 @@ private[unsafe] abstract class IORuntimeCompanionPlatform { this: IORuntime.type
       })
   }
 
+  @deprecated("bincompat shim for previous default method overload", "3.3.13")
+  def createDefaultComputeThreadPool(
+      self: () => IORuntime,
+      threads: Int,
+      threadPrefix: String): (WorkStealingThreadPool, () => Unit) =
+    createDefaultComputeThreadPool(self(), threads, threadPrefix, DefaultBlockerPrefix)
```

Comment:
I'm surprised if this doesn't work, but whatever :)
```suggestion
    createDefaultComputeThreadPool(self(), threads, threadPrefix)
```